tests/src/test/scala/org/json4s/JsonFormatsSpec.scala

```diff
@@ -10,6 +10,9 @@ class JacksonJsonFormatsSpec extends JsonFormatsSpec[JValue]("Jackson") with jac
 * System under specification for JSON Formats.
 */
 abstract class JsonFormatsSpec[T](mod: String) extends Specification with TypeHintExamples with JsonMethods[T] {
+  // To ensure the state of the ObjectMapper is guaranteed, execute in order
+  sequential
+
```

Comment:
This looks like working around some problems than solving them.
Although I admit to have the tests more isolated is more of a hassle here.
So that's pragmatic approach. I am not against

Reply:
Certainly, this is a workaround.

I think the following PR may be resolved when it is merged, please review it too.

https://github.com/json4s/json4s/pull/711